Add callout box about tensorflow (#226)
carpentries-incubator/deep-learning-intro: _episodes/03-monitor-the-model.md

> {:.solution}
{:.challenge}

> ## Tensorboard
>

> > If we run many different experiments with different architecture,

Reviewer: ```suggestion
> If we run many different experiments with different architectures,
```

> > it can be difficult to keep track of and compare the performance of the different models.

Reviewer: ```suggestion
> it can be difficult to keep track of these different models or compare the achieved performance.
```

> We can use *tensorboard*, a framework that keeps track of our experiments and shows graphs like we plotted above.
> Tensorboard is included in our tensorflow installation by default.

> > We first need to add a *callback* to our (compiled) model that saves the training performance in a logs directory:

Reviewer: ```suggestion
> To use it, we first need to add a *callback* to our (compiled) model that saves the progress of training performance in a logs directory:
```

> ~~~
> from tensorflow.keras.callbacks import TensorBoard
> import datetime
> log_dir = "logs/fit/" + datetime.datetime.now().strftime("%Y%m%d-%H%M%S") # You can adjust this to add a more meaningful model name
> tensorboard_callback = TensorBoard(log_dir=log_dir, histogram_freq=1)
> history = model.fit(X_train, y_train,
>                     batch_size = 32,
>                     epochs = 200,
>                     validation_data=(X_val, y_val),
>                     callbacks=[tensorboard_callback],
>                     verbose = 2)
> ~~~
> {: .language-python}
> You can launch the tensorboard interface from a Jupyter notebook, showing all trained models:
> ~~~
> %load_ext tensorboard
> %tensorboard --logdir logs/fit
> ~~~
> {: .language-python}

> >

Reviewer: Might be worthwhile to add a screenshot of tensorboard, so that people have a confirmation on what we mean.
